src/components/SingletonEditor.tsx

```diff
@@ -41,6 +42,8 @@ export const SingletonDesignContext = createContext<ISingletonDesignContext>({
   editorState: { isLoaded: false, unlayer: undefined },
 });
 
+const AUTO_SAVE_INTERVAL = 6000;
```

Comment:
What do you think, at least for the beginning, to increase this value? What do you think about 30 segs?

Reply:
I agree!!